File: 1. Two Sum/main.py
class Solution(object):
    def twoSum1(self, nums, target):
        for n in range(len(nums)):
            for m in range(len(nums)-n):
                if n != m:
                    if nums[n] + nums[m] == target:
                        return [n, m]
                    else:
                        continue

    def twoSum2(self, nums, target):
        seen = {}
        for i in range(len(nums)):
            seen[nums[i]] = i

        for i in range(len(nums)):
            y = target - nums[i]

            if y in seen and seen[y] != i:
                return [i, seen[y]]

File: 1. Two Sum/main.js
function twoSum1(nums, target) {
  for (let n = 0; n < nums.length; n++) {
    for (let m = 0; m < nums.length - n; m++) {
      if (n !== m) {
        if (nums[n] + nums[m] === target) {
          return [n, m];
        }
      }
    }
  }
}
